pkg/k8shandler/configuration_tmpl.go

```diff
@@ -41,6 +41,9 @@ openshift.searchguard:
 
 openshift.kibana.index.mode: {{.KibanaIndexMode}}
 
+prometheus:
```

Comment:
do we not have a `_test` file to go along with this?

Reply:
I didn't find any (and tests are passing). May be @vimalk78 can confirm?